bspump/cache/memcached.py

```diff
@@ -0,0 +1,47 @@
+from pymemcache.client import base
+import subprocess
+import logging
+
+L = logging.getLogger(__name__)
+
+
+class Memcached:
+	"""
+	Memcached for use in bspump.
+	Parameters:
+		memcache - ip:port of memcache service
+	"""
+	def __init__(self, app, max_size=1000, expiry_seconds=0, memcache=None *args, **kwargs):
+		super().__init__(*args, **kwargs)
+		# TODO: Configurable via configfile?
+		if memcache:
+			self.Client = base.Client(tuple(memcache.split(":")))
+		else:
+			L.error("Memcache service path not set.")
+
+		self.App = app
+		self.MaxSize = max_size
+		self.Expiration = expiry_seconds
+
+
+	def set(self, to_chache: dict):
+		for key, value in to_chache:
+			returned = self.Client.set(key, value, expire=self.Expiration)
+		if returned is not True:
+			L.warning("Setting a memcached key-value failed")
+		L.debug("Successfully set key-value")
+
+	def get(self, key):
```

Comment:
This is so thin layer (class) that is it actually not needed.
This is one more reason to build it as a lookup or connection.

Reply:
Ok, sounds legit, will transform it into lookup.